modules/javafx.graphics/src/main/java/javafx/print/JobSettings.java

```diff
@@ -462,6 +462,98 @@ public void setJobName(String name) {
     }
     ///////////////////////  END JOBNAME /////////////////////
 
+    ///////////////////////  START OUTPUTFILE /////////////////////
+
+    private SimpleStringProperty outputFile;
+
+    /**
+     * A {@code StringProperty} representing the
+     * name of a filesystem file, to which the platform printer
+     * driver should spool the rendered print data.
+     * <p>
+     * Applications can use this to programmatically request print-to-file
+     * behavior where the native print system is capable of spooling the
+     * output to a filesystem file, rather than the printer device.
+     * <p>
+     * This is often useful where the printer driver generates a format
+     * such as Postscript or PDF, and the application intends to distribute
+     * the result instead of printing it, or for some other reason the
+     * application does not want physical media (paper) emitted by the printer.
+     * <p>
+     * The default value is an empty string, which is interpreted as unset,
+     * equivalent to null, which means output is sent to the printer.
+     * So in order to reset to print to the printer, clear the value of
+     * this property by setting it to null or an empty string.
```

Comment:
This doesn't flow as well as it could. I think you only need to mention once that `null` is the same as an empty string, and then you can just say "empty string". Maybe something like this?

```
The default value is an empty string, which means output is sent to the printer. So in order to reset
to print to the printer, clear the value of this property by setting it to an empty string. A value
of {@code null} is treated as an empty string.
```

Reply:
But I don't say it twice. I say
>  The default value is an empty string, which is interpreted as unset, equivalent to null,

and 
>  clear the value of this property by setting it to null or an empty string.

which is somewhat different and makes i t very clear that either will work .. 

So I think this is all fine and flows as intended.

Reply:
ok